From 243b778d84e1983a4ae5529469c978bca8988e72 Mon Sep 17 00:00:00 2001
From: PulkoMandy <pulkomandy@pulkomandy.tk>
Date: Tue, 1 Jul 2025 17:49:25 +0200
Subject: Remove -mcpu=native

It would prevent people with old machines to use rasm built on newer
ones.

diff --git a/makefile.MacOS b/makefile.MacOS
index cbd5787..c62822f 100644
--- a/makefile.MacOS
+++ b/makefile.MacOS
@@ -1,7 +1,7 @@
 CC=gcc
 EXEC=rasm
 
-CFLAGS=-lm -mcpu=native -o $(EXEC) -w
+CFLAGS=-lm -o $(EXEC) -w
 CFLAGS_OPT = $(CFLAGS) -O2
 CFLAGS_DBG = $(CFLAGS) -g -pthread -DRDD
 
-- 
2.48.1

